packages/components/src/custom-select-control-v2/styles.ts

```diff
@@ -105,10 +105,21 @@ export const Select = styled( Ariakit.Select, {
 } );
 
 export const SelectPopover = styled( Ariakit.SelectPopover )`
+	display: flex;
+	flex-direction: column;
+
+	background-color: ${ COLORS.theme.background };
 	border-radius: 2px;
-	background: ${ COLORS.theme.background };
 	border: 1px solid ${ COLORS.theme.foreground };
 
+	/* Same as popover component */
+	/* TODO: is there a way to read the sass variable? */
+	z-index: 1000000;
```

Comment:
Not at the moment (the build process isn't set up for it), but maybe we can add a comment for searchability?

```suggestion
	// z-index(".components-popover")
	z-index: 1000000;
```